config/routes.rb

```diff
@@ -1,12 +1,12 @@
 Rails.application.routes.draw do
-  get 'support', to: 'support#index'
+  get '/support', to: 'support#index'
```

Comment:
Could potentially set this just to `/` as opposed to `/support`? This would mean that is people go to the root `/` page they will be displayed one of your application pages as opposed to the 'welcome to rails' splash page or whatever the page would be when deployed. 

Can be done using 

```
root 'support#index' 
```
Unless you were using `/support` for particular reason or to make more explicit that they are on a support page?

Reply:
This was something I did want to change but the product page links to '/support', so for the time being I thought it would be best to keep the consistency between the two. But as the product page no longer links to the support form for the time being that is something I can now change.

Reply:
This has now been changed